### ISSUE TITLE
pre-release for Go 1.14

### ISSUE THREAD

**Issue by pierrre** [golangci/golangci-lint#962]
Hello,
Is it possible to have a pre-release (maybe a beta) for Go 1.14 ?
It seems that the pre-compiled binaries are not compatible with Go 1.14rc1.

**Comment by ernado** [comment]
Hey, I'm planning an alpha release compiled with go1.14rc1 until Friday.

**Comment by ernado** [comment]
Seems it is non-trivial to do it in automated way via our CI/CD.

**Comment by tpounds** [comment]
@ernado, @pierrre I've updated the `.travis.yml` to build against Go `1.14rc1` in #964. However, https://github.com/golangci/golangci-lint/blob/master/.travis.yml#L34 needs to be updated to to `1.14+` before new a release will be built that supports the version. @pierrre Have you tried installing via `go get` now that #930 has been merged?

**Comment by pierrre** [comment]
Thank you, but I have another very strange error:
```
ERRO Running error: failed to pre-run govet: failed to configure analyzers: settings key "printf" must be valid analyzer name, valid analyzers: [asmdecl assign atomic atomicalign bools buildtag cgocall composites copylocks deepequalerrors errorsas findcall httpresponse loopclosure lostcancel nilfunc nilness shadow shift sortslice stdmethods structtag tests unmarshal unreachable unsafeptr unusedresult]
```

My config:
```yaml
run:
  timeout: "10m"
linters:
  disable-all: true
  enable:
    - "bodyclose"
    - "deadcode"
    - "depguard"
    - "errcheck"
    - "gocritic"
    - "gocyclo"
    - "gofmt"
    - "goimports"
    - "golint"
    - "govet"
    - "ineffassign"
    - "megacheck"
    - "misspell"
    - "nakedret"
    - "structcheck"
    - "unconvert"
    - "unparam"
    - "varcheck"
linters-settings:
  depguard:
    list-type: blacklist
    include-go-root: true
    packages:
      - errors
      - reflect
      - unsafe
      - github.com/pkg/errors
  gocyclo:
    min-complexity: 10
  govet:
    enable-all: true
    settings:
      printf:
        funcs:
          - "(github.com/xxx/golang-libraries/errors).Newf"
          - "(github.com/xxx/golang-libraries/errors).WithMessagef"
          - "(github.com/xxx/golang-libraries/errors).Wrapf"
issues:
  exclude-use-default: false
  max-issues-per-linter: 0
  max-same-issues: 0
```

I think I've already mentioned this issue somewhere.
It's due to a recent change in x/tools (the config of the govet linter)

**Comment by ernado** [comment]
@pierrre what version is causing that error?

**Comment by pierrre** [comment]
v1.23.3 installed with `go get`
`cd /tmp && GO111MODULE=on GOBIN=/home/pierre/go/pkg/golangci-lint/v1.23.3-mod go get -v github.com/golangci/golangci-lint/cmd/golangci-lint@v1.23.3`

But I'm dumb: I understand now that #930 has not yet been released in a tag, correct ?

**Comment by ernado** [comment]
Oh, I forgot to mention that, sorry :(
You can try version from my [comment](https://github.com/golangci/golangci-lint/issues/962#issuecomment-583733469) or build it by yourself with `goreleaser --snapshot` command in project root.

**Comment by ernado** [comment]
I've also issued `v1.23.4` that will be available in ~20min.

**Comment by ernado** [comment]
Meh, due to some issues with CI/CD the `v1.23.6` is valid latest version.

**Comment by pierrre** [comment]
The install with `go get` and `v1.23.6` works perfectly fine for me now !
Thank you very much!

Do you plan to make this installation method supported again ?

**Comment by ernado** [comment]
The `go get` method should work until we have any critical `replace` directives and go tools are not updating in backwards incompatible manner (e.g. last bug with x/tools).

Personally I will try to fix any go-get related bugs, so it is kinda supported, but binary releases are still recommended.

**Comment by pierrre** [comment]
OK.

I will close this issue now, because my problem is fixed: I can use golangci-lint with Go 1.14rc1.

FYI, I'm using this command to install it:
```
cd /tmp && GO111MODULE=on GOBIN=/home/pierre/go/pkg/golangci-lint/v1.23.6-mod go get -v github.com/golangci/golangci-lint/cmd/golangci-lint@v1.23.6
```

**Comment by pierrre** [comment]
It's very strange:
- my local Go version is 1.14rc1
- I'm running golangci-lint v1.23.6 from the **pre-compiled** binaries you provide
- it works without any error

So, it seems I don't need to use `go get` :thinking: 
The v1.23.6 compiled with Go 1.13 works with my local Go 1.14rc1
Am I misunderstanding something ?

**Comment by ernado** [comment]
Seems like there is no hard requirement to use go1.14 to support go1.14, that works for mee too 👀

**Comment by ldez** [comment]
go1.14 is out :tada: 

https://twitter.com/golang/status/1232421783548067840?s=21
